build/Dockerfile.platform-validation

```diff
@@ -0,0 +1,23 @@
+FROM quay.io/sustainable_computing_io/kepler_builder:ubi-9-libbpf-1.2.0-go1.18 as builder
```

Comment:
The validator image repo is created, please update the push image process to push it to this image:
podman pull quay.io/sustainable_computing_io/kepler-validator

Reply:
> The validator image repo is created, please update the push image process to push it to this image: `quay.io/sustainable_computing_io/kepler-validator`

How could I test this before updating the PR? Since such push operation needs dependencies like [here](https://github.com/sustainable-computing-io/kepler/blob/main/.github/workflows/image.yml#L23-L26)